File: Admin-MusicStore(RESTful API)/src/main/java/com/example/demo/domain/Album.java
package com.example.demo.domain;


public class Album {
	private int AlbumId ;
	private String Title ;
	private double Price ;
	private String AlbumArtUrl;
	private Genre genre ; 
	private Artist artist;
	
	public Album(){}

	public Album(int albumId, String title, double price, String albumArtUrl, Genre genre, Artist artist) {
		super();
		AlbumId = albumId;
		Title = title;
		Price = price;
		AlbumArtUrl = albumArtUrl;
		this.genre = genre;
		this.artist = artist;
	}

	public int getAlbumId() {
		return AlbumId;
	}

	public void setAlbumId(int albumId) {
		AlbumId = albumId;
	}

	public String getTitle() {
		return Title;
	}

	public void setTitle(String title) {
		Title = title;
	}

	public double getPrice() {
		return Price;
	}

	public void setPrice(double price) {
		Price = price;
	}

	public String getAlbumArtUrl() {
		return AlbumArtUrl;
	}

	public void setAlbumArtUrl(String albumArtUrl) {
		AlbumArtUrl = albumArtUrl;
	}

	public Genre getGenre() {
		return genre;
	}

	public void setGenre(Genre genre) {
		this.genre = genre;
	}

	public Artist getArtist() {
		return artist;
	}

	public void setArtist(Artist artist) {
		this.artist = artist;
	}

	
}

File: Admin-MusicStore(Angular)/src/app/domain/album.ts
import { Artist } from '../domain/artist';
import { Genre } from '../domain/genre';
import { Injectable }    from '@angular/core';

export class Album{
  public  albumId: number;
  public title: string;
  public price: number;
  public albumArtUrl : string;
  public genre : Genre;
  public artist : Artist;

  constructor(albumId: number, title: string, price: number, albumArtUrl : string, genre : Genre, artist : Artist){
     this.albumId = albumId;
      this.title = title;
      this.price = price;
      this.albumArtUrl = albumArtUrl;
      this.genre = genre ;
      this.artist = artist;
   }

  // constructor(albumId: number, title: string, price: number, albumArtUrl : string, genreId : number
  //   ,genreName : string,artistId : number, artistName : string){
  //    this.albumId = albumId;
  //     this.title = title;
  //     this.price = price;
  //     this.albumArtUrl = albumArtUrl;
  //     this.genre.genreId = genreId ;
  //     this.genre.genreName = genreName;
  //     this.artist.artistId = artistId;
  //     this.artist.artistName = artistName;
  //  }

}
